spec/mongoid/extensions/big_decimal_spec.rb

```diff
@@ -273,6 +273,17 @@
         end
       end
 
+      context "when the value is castable" do
+
+        let(:value) do
+          2.hours
+        end
+
```

Comment:
Could you add a `before` here please asserting the type of `value` to be the `Duration`?

Reply:
done